docs/monitors/gitlab.md

```diff
@@ -54,7 +54,7 @@ metrics are just targets `gitlab_monitor_database`,
 
 | Agent Monitor Type    |     Gitlab Doc                           | Standard Port | Standard Path |
 |-----------------------|------------------------------------------|---------------|---------------|
-| gitlab | [Gitlab doc](https://docs.gitlab.com/ee/administration/monitoring/prometheus/gitlab_monitor_exporter.html) | 9168 | /metrics |
+| gitlab | [Gitlab doc](https://docs.gitlab.com/ee/administration/monitoring/prometheus/gitlab_exporter.html) | 9168 | /metrics |
```

Comment:
@asuresh4 can you split out this Gitlab doc fixes to another PR?

Reply:
@keitwb the gitlab doc is fixed in master. I cherry-picked here. Maybe we should rebase `k8s-node-uid` with latest from master?